person.rb

```diff
@@ -1,9 +1,16 @@
 # Class that represents a person
-class Person
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'correct_name method must be implemented in a subclass.'
+  end
+end
```

Comment:
- [ ] It is against best practices to have multiple classes in one file. Please create separate files for all your classes and 'require' them when needed. This will help you have a cleaner and well-organized code structure 👍